src/sass/nft-collection-box.scss

```diff
@@ -0,0 +1,84 @@
+.element {
+  position: relative;
+  min-width: 200px;
+  .header {
```

Comment:
Since the scss is scoped, there is no need to embed `.header .image .content and .footer` inside of `.element` theses can be moved to the same level as `.element` doing so will make the final css names much shorter and save on kb space.